packages/web-app/components/dispute/DisputeActions.tsx

```diff
@@ -14,9 +14,9 @@ export const DisputeArbitrationActions = () => {
 	if (mode == "edit") {
 		return <ResolutionForm />;
 	} else if (resolution == undefined) {
-		return <Button label="Submit a resolution" onClick={() => setMode("edit")} />;
+		return <Button label="Propose a settlement" onClick={() => setMode("edit")} />;
```

Comment:
I think this is the step we said to add the case for "Propose another settlement" if there is already one ongoing.
<img width="415" alt="image" src="https://user-images.githubusercontent.com/60603572/210394606-97a06a08-ac6d-45d8-a1d0-fd23411d9eb6.png">

Could you confirm @xPi2

Reply:
Yes, we should add an extra button when there is already proposed settlements.